src/helpers.ts

```diff
@@ -0,0 +1,27 @@
+import moment from "moment";
+
+export const getDateTimeString = (type:string, date:Date) : string => {
+  const d = moment(date).utc(false);
+
+  if (type === 'start'){
+    d.hours(0);
+    d.minutes(0);
+    d.seconds(0);
+    d.milliseconds(0);
+  }
+
+  if (type === 'end' && !isDateToday(date)){
+    d.hours(23);
+    d.minutes(59);
+    d.seconds(59);
+    d.milliseconds(999);
+  }
+
+  return d.toISOString();
+}
+
+const isDateToday = (date:Date):boolean => {
```

Comment:
Could easily perfom the task via a oneliner.
```suggestion
const isDateToday = (date:Date):boolean => new Date().getTime() - date.getTime() < 8.64e+7;
```